src/app/pages/charging-stations/charging-station/ocpp-parameters/charging-station-ocpp-parameters-editable-table-data-source.component.ts

```diff
@@ -45,12 +41,13 @@ export class ChargingStationOcppParametersEditableTableDataSource extends Editab
       hasDynamicRowAction: true,
       search: {
         enabled: true
-      }
+      },
+      hasPristineFormAtLoad: true
     };
   }
 
   public buildTableActionsDef(): TableActionDef[] {
-    return [new TableExportAction().getActionDef()];
+    return [new TableExportAsCSVAction().getActionDef()];
```

Comment:
As this is specific to OCPP, I would expect this naming: TableExportOCPPAsCSVAction